TriblerGUI/tribler_window.py

```diff
@@ -949,6 +946,26 @@ def clicked_force_shutdown(self):
         # Stop the Qt application
         QApplication.quit()
 
+    def clicked_skip_conversion(self):
+        self.dialog = ConfirmationDialog(self, "Abort the conversion of old channels",
+                                         "Upgrade procedure is now converting channels data "
```

Comment:
Please start the sentence with: "The upgrade procedure..."

Channels data -> torrents in channels

Reply:
OK